tests/blackbox_test.go

```diff
@@ -311,3 +338,24 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 	}
 	return nil
 }
+
+//Identify and replace $uuid<num> with correct UUID
```

Comment:
nit: we typically leave a space after `//`. E.g: `// Identify...` instead of `//Identify...`